apps/antalmanac/src/routes/Home.tsx

```diff
@@ -12,13 +12,17 @@ import NotificationSnackbar from '$components/AppBar/NotificationSnackbar';
 
 export default function Home() {
     const isMobileScreen = useMediaQuery('(max-width: 750px)');
+
     const theme = useTheme();
 
     return (
         <MuiPickersUtilsProvider utils={DateFnsUtils}>
             <CssBaseline />
+
             <PatchNotes />
-            <Bar />
+
+            <Header />
+
             {isMobileScreen ? (
                 <MobileHome />
             ) : (
```

Comment:
Not sure why, but the app isn't seamlessly swapping between Mobile and Desktop views anymore when the schedule has unsaved courses.

![Resizing to Mobile creates warning(https://github.com/icssc/AntAlmanac/assets/100006999/1e75004d-b465-441e-8842-643cafd3a31d)

Reply:
It was switching between two components and both were loading the schedule automatically when mounting. I created a store to keep track of the first time the app ever attempts to load the schedule automatically, and check that before automatically loading.